For what it's worth, [shellcheck](https://github.com/koalaman/shellcheck/wiki/SC2143) isn't happy with this change. But i guess the output of lsmod is not too bad for this to really matter.

--- a/tooling/bundler/src/bundle/linux/templates/appimage
+++ b/tooling/bundler/src/bundle/linux/templates/appimage
@@ -34,7 +34,8 @@ chmod +x linuxdeploy-plugin-gtk.sh
 chmod +x linuxdeploy-x86_64.AppImage
 
 args=()
-if ! [[ $(lsmod | grep fuse) ]]; then
+if ! lsmod | grep -q 'fuse'
+then
   args+=( '--appimage-extract-and-run' )
 fi
 

